packages/orchestration/src/index.ts

```diff
@@ -47,7 +47,10 @@ export type {
 
 export { OrchestrationClient } from './orchestration-client.js';
 
-export { buildAzureContentFilter } from './orchestration-utils.js';
+export {
+  buildAzureContentFilter,
+  ContentFilters
```

Comment:
[req] The azure function should also be exported here.

Reply:
For me this a nice to have.

Reply:
If we do export it, we also need to add more docs on it though.

Reply:
[req] Thinking about this further, ContentFilters.azure() actually isn't following our style-guide. So that need's to be considered in the final naming..

About different namings, I'm afraid that might lead to more confusion. 
Agreed that the 2 import behaviors should be documented

Reply:
I have renamed `azure` into `azureContentSafety` because for me it is also a bit weird later to call llama guard just llama. Long naming I guess it is.